modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/js/Attachment/AttachmentLocalizedObjectField.tsx

```diff
@@ -0,0 +1,167 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {ClayInput} from '@clayui/form';
+import {
+	EditingLocale,
+	LocalesDropdown,
+} from 'dynamic-data-mapping-form-field-type';
+import {
+	getEditingLocales,
+	getLocale,
+} from 'dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/localizedObjectFields/util/locales';
+import {
+	FieldChangeEventHandler,
+	LocalizedValue,
+} from 'dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/types';
```

Comment:
I think we don't need to reference the full path because these interfaces are exported in [index.ts of this module](https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/index.ts#L105).

Check examples [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/js/Attachment/Attachment.tsx#L15) and [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/js/MultiSelectPicklist/MultiSelectPicklist.tsx#L9).